s3file/static/s3file/js/s3file.js

```diff
@@ -101,6 +139,7 @@
   }
 
   document.addEventListener('DOMContentLoaded', () => {
+    s3ProgressDiv = document.getElementById("s3UploadFileProgress");
```

Comment:
You are assuming here there's only a single form on the site ;)

Reply:
fixed this by adding a selector that selects a child node of the current form,  so this would change the HTML slightly to need to have the bar nested within the form element,  for example
```html
    <form method="post" class="form" enctype="multipart/form-data" action="{% url 'idtr-ui-create-job' %}" onsubmit='toggleCreateButton(false); '>
          {% bootstrap_form form_upload %}
          {% buttons %}
            <a href="{% url 'idtr-ui-jobs' %}" class="btn btn-link" role="button">Cancel</a>
            <button id='createButton' type="submit" class="btn btn-primary" onclick="toggleProcessing(true, false);">Create</button>
          {% endbuttons %}
          <div id="s3UploadFileProgressBar" class="progress" style="height: 20px;">
            <div id="s3UploadFileProgress" class="progress-bar progress-bar-striped" role="progressbar" style="width:0%;" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100"></div>
          </div>
        </form>
```

Let me know your thoughts on this approach,  this would at least require the `id` being within the form (and would leave the opportunity to have the same progressbar `id` in other forms on the page,  which I've tested and works)